apps/protocol-api/src/prisma/schema.prisma

```diff
@@ -83,8 +84,11 @@ model GuildUser {
   user User @relation(fields: [user_id], references: [id])
   guild_id Int
   guild Guild @relation(fields: [guild_id], references: [id])
+  membership_status_id Int
+  membershipStatus GuildMembershipStatus @relation(fields: [membership_status_id], references: [id])
+  favorite Boolean @default(false)
 
-  @@unique([user_id, guild_id])
+  @@unique([user_id, guild_id, membership_status_id])
```

Comment:
The same member can have different statuses in a guild? I don't  think we want to support that use vase